jinahub/indexers/merger/SimpleMerger/simple_merger.py

```diff
@@ -0,0 +1,34 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from collections import OrderedDict
+from typing import List, Tuple
+
+from jina import Executor, requests, DocumentArray
+
+
+class SimpleMerger(Executor):
+    """
+    The SimpleMerger merges the results of shards by appending all matches..
+
+    :param default_traversal_paths: traverse path on docs, e.g. ['r'], ['c']
+    :param args: additional arguments
+    :param kwargs: additional key value arguments
+    """
+
+    def __init__(self, default_traversal_paths: Tuple[str] = ('r',), **kwargs):
+
+        self.default_traversal_paths = default_traversal_paths
```

Comment:
why putting it before `super().__init__(**kwargs). It's the same but looks weird

Reply:
thanks, I moved it